src/navigator/NavigatorWrapper.tsx

```diff
@@ -73,6 +74,9 @@ export const NavigatorWrapper = () => {
 
   const dispatch = useDispatch()
 
+  // @ts-ignore using a v6 navigation plugin with v5 types
```

Comment:
could we solve this by using v5 of the devtools instead?

Reply:
It looks like there's some issues with how the navigationRef is setup using the v5 version. I'd rather wait until https://github.com/valora-inc/wallet/pull/3004 is merged then resolve any issues rather than trying to find a way to make v5 work only to update.

Reply:
The Flipper plugin is only available in v6 of [@react-navigation/devtools](https://github.com/react-navigation/react-navigation/tree/main/packages/devtools). It might make sense to merge https://github.com/valora-inc/wallet/pull/3004. Then come back to this.